deploy/vulnerability/cloudbeat-vuln-mgmt.yml

```diff
@@ -0,0 +1,53 @@
+cloudbeat:
+  config:
+    v1:
+      type: vuln_mgmt
```

Comment:
should not we also have aws entry like in [cspm aws](https://github.com/elastic/cloudbeat/blob/main/deploy/aws/cloudbeat-aws.yml#L5)?

Reply:
In vuln-mgmt we don't use this section but I added the `deployment: aws` as in https://github.com/elastic/integrations/blob/e6b008862e4806ecaa1ac8a4ed80a90a6d34b497/packages/cloud_security_posture/data_stream/vulnerabilities/agent/stream/aws.yml.hbs#L4